tools/run-task

```diff
@@ -132,25 +113,22 @@ function listrify(steps, {concurrent = false, verbose = false} = {}) {
             if (Array.isArray(task)) {
                 return { title, task: () => listrify(task.map(_task => {
```

Comment:
Ah right, I see the purpose of `listrify` now, it looks like it's doing the right thing 😄